finagle-core/src/test/scala/com/twitter/finagle/loadbalancer/TrafficDistributorTest.scala

```diff
@@ -68,8 +68,9 @@ private object TrafficDistributorTest {
     override def toString: String = s"Balancer($endpoints)"
   }
 
-  // Return the distribution for the the given `balancer` as a tuple
-  // of (weight, size, offer load).
+  /** Return the distribution for the the given `balancer` as a tuple
```

Comment:
can we start the sentence that begins with "Return" on the next line?  I think that's the scaladoc style we normally use.

Reply:
Will do